crates/pop-contracts/src/templates.rs

```diff
@@ -0,0 +1,142 @@
+// SPDX-License-Identifier: GPL-3.0
+use strum::{
+	EnumMessage as EnumMessageT, EnumProperty as EnumPropertyT, VariantArray as VariantArrayT,
+};
+use strum_macros::{AsRefStr, Display, EnumMessage, EnumProperty, EnumString, VariantArray};
+
+use crate::errors::Error;
+
+#[derive(
+	AsRefStr,
+	Clone,
+	Debug,
+	Default,
+	Display,
+	EnumMessage,
+	EnumProperty,
+	EnumString,
+	Eq,
+	Hash,
+	PartialEq,
+	VariantArray,
+)]
+pub enum Template {
+	/// A minimalist contract template.
+	#[default]
+	#[strum(serialize = "standard", message = "Standard", detailed_message = "Our 'Hello World")]
+	Standard,
+	/// The implementation of the ERC-20 standard in ink!
+	#[strum(
+		serialize = "erc20",
+		message = "Erc20",
+		detailed_message = "The implementation of the ERC-20 standard in ink!",
+		props(Repository = "https://github.com/paritytech/ink-examples")
+	)]
+	ERC20,
+	/// The implementation of the ERC-721 standard in ink!
+	#[strum(
+		serialize = "erc721",
+		message = "Erc721",
+		detailed_message = "The implementation of the ERC-721 standard in ink!",
+		props(Repository = "https://github.com/paritytech/ink-examples")
+	)]
+	ERC721,
+	/// The implementation of the ERC-1155 standard in ink!
+	#[strum(
+		serialize = "erc1155",
+		message = "Erc1155",
+		detailed_message = "The implementation of the ERC-1155 standard in ink!",
+		props(Repository = "https://github.com/paritytech/ink-examples")
```

Comment:
Please update all links to use https://github.com/use-ink/ink-examples

Also, (per my longer comment above), having an additional category would be nice to structure all ERCs together, all PSPs together, etc.